Updates/0114_shattered_sun_trainee_event.sql

```diff
@@ -0,0 +1,305 @@
+(2514115, 0, 1, 22, 0, 0, 0, 0, 4, 0, "Commander Steele shouts at recruits"),
+(2514115, 0, 35, 6, 20, 0, 0, 0, 0, 0, "Commander Steele makes recruits turn 90° Counter-Clockwise"),
+(2514115, 27000, 1, 22, 0, 0, 0, 0, 4, 0, "Commander Steele shouts at recruits"),
+(2514115, 27000, 35, 5, 20, 36, 0, 0, 0, 0, "Commander Steele makes recruits attack"),
+(2514115, 29000, 35, 5, 20, 43, 0, 0, 0, 0, "Commander Steele makes recruits block"),
+(2514115, 32000, 35, 5, 20, 60, 0, 0, 0, 0, "Commander Steele makes recruits kick"),
+(2514117, 0, 1, 22, 0, 0, 0, 0, 4, 0, "Commander Steele shouts at recruits"),
+(2514117, 0, 35, 5, 20, 36, 0, 0, 0, 0, "Commander Steele makes recruits attack"),
+(2514118, 0, 1, 22, 0, 0, 0, 0, 4, 0, "Commander Steele shouts at recruits"),
+(2514118, 0, 35, 5, 20, 60, 0, 0, 0, 0, "Commander Steele makes recruits kick"),
+(2514118, 0, 20, 2, 2, 0, 19216, 150, 7, 0, "Almonen starts moving to the recruit-event"),
+(2514119, 0, 1, 22, 0, 0, 0, 0, 4, 0, "Commander Steele shouts at recruits"),
+(2514119, 0, 35, 5, 20, 43, 0, 0, 0, 0, "Commander Steele makes recruits block"),
+(2514121, 0, 1, 22, 0, 0, 0, 0, 4, 0, "Commander Steele shouts at recruits"),
+(2514121, 0, 35, 5, 20, 36, 0, 0, 0, 0, "Commander Steele makes recruits attack"),
+(2514122, 0, 1, 22, 0, 0, 0, 0, 4, 0, "Commander Steele shouts at recruits"),
+(2514122, 0, 35, 5, 20, 60, 0, 0, 0, 0, "Commander Steele makes recruits kick"),
+(2514124, 4000, 1, 253, 0, 0, 0, 0, 4, 0, "Commander Steele applauds"),
+(2514124, 6000, 35, 5, 20, 66, 0, 0, 0, 0, "Commander Steele makes recruits salute"),
+(2514124, 6000, 1, 30, 0, 0, 0, 0, 4, 0, "Commander Steele stops applauding"),
+(2514125, 0, 28, 8, 0, 0, 0, 0, 4, 0, "Commander Steele kneels at Almonen's speech"),
+(2514126, 0, 28, 0, 0, 0, 0, 0, 4, 0, "Commander Steele stands up"),
+(2514126, 0, 1, 253, 0, 0, 0, 0, 4, 0, "Commander Steele applauds"),
+(2514126, 9000, 1, 30, 0, 0, 0, 0, 4, 0, "Commander Steele stops applauding"),
+(2514127, 0, 0, 0, 0, 0, 0, 0, 4, 16448, "Listen up Men!"),
+(2514127, 4000, 0, 0, 0, 0, 0, 0, 4, 16449, "Your training is now complete. You each have your assignments. May the Light guide you."),
+(2514127, 9900, 35, 5, 25, 30, 0, 0, 0, 0, "Recruits stop applauding"),
+(2514127, 10000, 0, 0, 0, 0, 0, 0, 4, 16446, "Troops! Fall out!"),
+(2514127, 10000, 35, 9, 0, 0, 0, 0, 4, 0, "Commander Steele sends off recruits"),
+(2514130, 0, 1, 2, 0, 0, 0, 0, 4, 0, "Commander Steele bows to Lord Torvos"),
+(2514130, 3000, 0, 0, 0, 0, 0, 0, 4, 16401, "DIALOGUE-REPLACEME - 16401"),
+(2514130, 22000, 0, 0, 0, 0, 0, 0, 4, 16403, "DIALOGUE-REPLACEME - 16403"),
+(2514130, 28000, 0, 0, 0, 0, 0, 0, 4, 16404, "DIALOGUE-REPLACEME - 16404"),
+(2514130, 38000, 0, 0, 0, 0, 19202, 5, 7, 16412, "DIALOGUE-REPLACEME(Mordin's response) 16412"),
+(2514130, 66000, 0, 0, 0, 0, 19202, 5, 7, 16413, "DIALOGUE-REPLACEME(Mordin's response) 16413"),
+(2514130, 81000, 0, 0, 0, 0, 0, 0, 4, 16405, "DIALOGUE-REPLACEME - 16405"),
```

Comment:
will get proper comments when missing data becomes available

Reply:
Did you find something for this?